service_container/autowiring.rst

```diff
@@ -363,9 +363,41 @@ If you register this as a service, you now have *two* services that implement th
 which one to use. Remember, autowiring isn't magic; it looks for a service
 whose id matches the type-hint. So you need to choose one by creating an alias
 from the type to the correct service id (see :ref:`autowiring-interface-alias`).
+Additionally, you can define several named aliases if you want to use
+one implementation in some cases, and another implementation in some
+other cases.
 
-If you want ``Rot13Transformer`` to be the service that's used for autowiring, create
-that alias:
+
+
+For instance, you may want to use by default the ``Rot13Transformer``
+implementation when the ``TransformerInterface`` interface is type
+hinted, but use the ``UppercaseTransformer`` implementation in some
```

Comment:
I suggest you add a slight example of a class needing `TransformerInterface` and then explain the use-case with `Rot13Transformer` and `UppercaseTransformer`, for more conciseness

Reply:
There is already such a class in the previous code (`TwitterClient`), and `Rot13Transformer` and `UppercaseTransformer` are both existing classes used as examples previously, I don't think they require more explanation than they did before this PR, do they?